lib/index.js

```diff
@@ -175,7 +169,7 @@ export function configureUnifiedLanguageServer(
           streamError: new PassThrough(),
           streamOut: new PassThrough()
         },
-        (error, code, context) => {
+        (error, _, context) => {
```

Comment:
I prefer using a proper name for variables, even if they are unused.

Reply:
Main reason for me is that TS in my editor is info’ing about it being unused otherwise.

I’d be 🤷‍♂️ otherwise